langchain-core/src/language_models/base.ts

```diff
@@ -216,7 +216,7 @@ export interface FunctionDefinition {
    * To describe a function that accepts no parameters, provide the value
    * `{"type": "object", "properties": {}}`.
    */
-  parameters: Record<string, unknown>;
+  parameters?: Record<string, unknown>;
```

Comment:
Core changes should be done in a separate PR

Reply:
Should I open a new PR with that singular change that gets merged before this one?

Reply:
Yes, if you wouldn't mind that would be really helpful!